### PR TITLE
[ADD] hr_cae_event: P1 - Inscription à une infocoll : nouveaux champs

### PR DESCRIPTION
**Pull request by robinkeunen**

### CODE REVIEW

**Comment by mclaeysb** [comment]
Force-pushed after rebasing on `hr_cae_applicant` and fixing conflicts. 
Added promotions from P11 task.
@robinkeunen, don't forget to rebase!

**Comment by robinkeunen** [comment]
Outfi, I think that might mess up my codebase that you rebased on the other branch. I'll see how it goes 😨

**Comment by mclaeysb** [comment]
Not sure why CI doesn't find `event/partner_event`?

**Comment by mclaeysb** [comment]
Fixed an error in `hr_cae_applicants` tests > CI is green again 💚

File: hr_cae_event/views/event.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        <field name="model">event.event</field>
+        <field name="inherit_id" ref="event.view_event_form"/>
+        <field name="arch" type="xml">
+
+            <field name="organizer_id" position="after">
+                <field name="department_id"/>
+            </field>
+
+            <field name="date_end" position="after">
+                <field name="duration"/>
```
Comment (mclaeysb) [inline hr_cae_event/views/event.xml]: I would add in the XML (and/or in the help) that the unit is hours.

**Review by mclaeysb — APPROVED** [review]
LGTM!

**Comment by robinkeunen** [comment]
merged manually

**Pull request closed without merging**